src/structured-data-tags.js

```diff
@@ -10,11 +10,60 @@ function ldJson(type, fields) {
   };
 }
 
-export function StructuredDataTags({structuredData = {}}, config, pageType, data, {url}) {
+function generateArticleData (story = {}, publisherConfig = {}){
+  const metaKeywords = story.seo && story.seo['meta-keywords'] || [];
+  const { authors = [] } = story;
+  const {themeConfig = {}} = publisherConfig["theme-attributes"];
+
+  return {
+    "author": authors.map(author => ({
+      "@type": "Person",
+      "givenName": author.name,
+      "name": author.name
+    })),
+    "keywords": metaKeywords,
+    "publisher": {
+      "@type": "Organization",
+      "name": publisherConfig['publisher-name'],
+      "logo": {
+        "@type": "ImageObject",
+        "url": themeConfig.logo ? themeConfig.logo : "https://quintype.com/logo.png",
```

Comment:
Do not default this. I think if publisher logo is missing, just do not send anything

Reply:
Also, this is a 404